### ISSUE TITLE
reduce_sum() function does not recognize 2nd argument 'axis'

### ISSUE THREAD

**Issue by kyungyunlee** [hunkim/DeepLearningZeroToAll#186]
There seems to be an issue in reduce_sum() function in lab-06-1-softmax_classifier.py

When the second argument is "axis=1", there is a TypeError saying "unexpected keyword argument."
From my experience with python, this shouldn't be an error, but I am not sure if it is a version issue(I have the most recent one though..) or something else. 

<img width="634" alt="screen shot 2017-06-30 at 11 41 35 am" src="https://user-images.githubusercontent.com/19829532/27718794-2aa61158-5d89-11e7-8af0-1ede03852d58.png">

In my case, it works  when I fix it from
`cost = tf.reduce_mean(-tf.reduce_sum(Y * tf.log(hypothesis), axis=1))` 
to 
`cost = tf.reduce_mean(-tf.reduce_sum(Y * tf.log(hypothesis), 1))`

Also a reference to tensorflow document.
[https://www.tensorflow.org/api_docs/python/tf/reduce_sum](https://www.tensorflow.org/api_docs/python/tf/reduce_sum)

**Comment by kkweon** [comment]
I'm not able to reproduce this error. You should be able to use the `axis` keyword.
Also, it's not a version issue because `axis` has been there since the beginning.

The only thing I suspect is that you claim there is an issue in the file `lab-06-1-softmax_classifier.py`
but your error message shows the error happens in `basics-01.py` is not relevant to this file. 

It seems like a sementic error to me.

**Comment by kyungyunlee** [comment]
I am very sorry about the file name and the confusion. I made my own file without cloning directly from git to try out different codes myself, so I was actually running the correct file `lab-06-1-softmax_classifier.py` and code. I also think that `axis` keyword should work, but it only works when written without...

Just wondering if it could help...I am using python3 with tensorflow '0.11.0rc2'
<img width="575" alt="screen shot 2017-07-03 at 12 52 02 am" src="https://user-images.githubusercontent.com/19829532/27771521-cc144160-5f8a-11e7-8757-ac2dba591c76.png">

Any idea/suggestion/lesson! would be appreciated!!
Thanks

**Comment by kkweon** [comment]
@kyungyunlee You are using too old version of tensorflow.
The current version is 1.2.1 and yours is 0.11.0 and even it's a rc(release candidate).

Please update your tensorflow first.

**Comment by kkweon** [comment]
Great!
